projects/packages/jetpack-mu-wpcom/src/features/newspack-blocks/synced-newspack-blocks/class-newspack-blocks.php

```diff
@@ -1252,7 +1253,7 @@ public static function disable_jetpack_donate() {
 
 		// Tell Jetpack to mark the donations feature as unavailable.
 		Jetpack_Gutenberg::set_extension_unavailable(
-			'donations',
+			'jetpack/donations',
```

Comment:
@kraftbj I tagged you for review, as you removed the prefix in #39734; in general this is a file that is synced as-is (see [here](https://github.com/Automattic/jetpack/blob/8468f0b5feb29e1a0b2d0b57376f814671abae58/projects/packages/jetpack-mu-wpcom/src/features/newspack-blocks/README.md?plain=1#L29)).

If we really don't want that prefix, we can update the script to target that specific string, though it would need to be pretty specific to avoid false positives.

On the other hand, I have no idea if this is something that should actually be disabled in this package's version...should we really be disabling the extension in favor of Newspack's variant, given we don't even sync that block? 🤔

Reply:
Thanks for this. The `set_extension_unavailable` still checks for the `jetpack/` prefix and strips it, so it works either way. It is, in theory, just slightly more performant since it can skip out of the prefix check early.

Without knowing how Newspack Blocks check for Jetpack version, we can leave upstream as it is (and thus accept this merge bringing back the prefix).